plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -885,6 +886,15 @@ TIMESTAMP_TZ_MILLIS, timestampWithTimeZoneDataType(3).toLiteral(timeGapInKathman
                 .execute(getQueryRunner(), oracleCreateAndInsert("timestamp_tz"));
     }
 
+    @Test
+    public void testNanoTimestamp()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("TIMESTAMP(9)", "TIMESTAMP '2019-03-18 10:01:17.123456111'",
+                        TIMESTAMP_NANOS, "TIMESTAMP '2019-03-18 10:01:17.123456111'")
```

Comment:
Could you add more test cases? min, min-1, max, negative, epoch, dst, julian->gregorian switch, max+1, null etc.

Reply:
@ebyhr tests added